horace_core/sqw/file_io/@sqw_file_interface/sqw_file_interface.m

```diff
@@ -125,7 +125,7 @@
         %-------------------------
         function obj = delete(obj)
             % destructor, which is not fully functioning
-            % operation for normal(non-handle) Matlab classes.
+            % operation for normal(non-handle) MATLAB classes.
             % Usually needs the class on lhs of delete expression or
             % understanding when this can be omitted
```

Comment:
This comment is confusing as it doesn't seem to relate the the operation itself, more complaining about issues with MATLAB at large. 
```suggestion
% Close files and invalidate object to prevent further use. 
% Not true destructor
```